src/view/RoundView.java

```diff
@@ -29,14 +33,25 @@ public class RoundView extends JPanel implements ActionListener, PropertyChangeL
     private final ToggleAudioViewModel toggleAudioViewModel;
     private final ToggleAudioController toggleAudioController;
     private final FinishRoundController finishRoundController;
-
+    private final FinishRoundViewModel finishRoundViewModel;
+    private final CreateGameViewModel createGameViewModel;
+    private final LoadGameViewModel loadGameViewModel;
     private final JButton playSong;
     private final JButton submit;
+    private final JButton easyButton1;
+    private final JButton easyButton2;
+    private final JButton mediumButton1;
+    private final JButton mediumButton2;
+    private final JButton mediumButton3;
+    private final JButton mediumButton4;
     private final JLabel roundInfo;
     private final JLabel livesInfo;
     private final JLabel genreInfo;
     private final JTextField answerInputField;
     private final JLabel loadingLabel;
+    private final JPanel easyAnswerSection;
+    private final JPanel mediumAnswerSection;
+    private final JPanel hardAnswerSection;
```

Comment:
```suggestion
    private final JLabel roundInfo;
    private final JLabel livesInfo;
    private final JLabel genreInfo;
    private final JTextField answerInputField;
    private final JLabel loadingLabel;
    private final JPanel answerSection;
```
We can delete the hard coded buttons and sections in favour of one answerSection that dynamically creates buttons depending on how many options there are at the time.